### PR TITLE
Add sample and update Getting started

### PR DESCRIPTION
**Pull request by qmhu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,49 +76,311 @@ Crane is composed of the following components:
...
+**Deploying Crane and Fadvisor**
+
+Download Crane and configuration prometheus
 
-helm install $RELEASE_NAME -n $NAMESPACE --set kubeStateMetrics.enabled=false --set pushgateway.enabled=false --set alertmanager.enabled=false --set server.persistentVolume.enabled=false --create-namespace  prometheus-community/prometheus
+```console
+git clone git@github.com:gocrane/crane.git
+cd ./crane
+git checkout v0.1.0
+sed -i '' "s/PROMETHEUS_ADDRESS/http:\/\/prometheus-server.crane-system.svc.cluster.local:8080/" deploy/craned/deployment.yaml
```
Comment (mfanjie) [inline README.md:118]: as we set crane-system as the default monitoring namespace by default, I would suggest change the template directly and replace PROMETHEUS_ADDRESS with `http://prometheus-server.crane-system.svc.cluster.local:8080/` and replace it by sed in advanced mode.

File: README.md [hunk 2]
```diff
@@ -76,49 +76,311 @@ Crane is composed of the following components:
...
 ```
 
-Crane use prometheus to be the default metric provider. Using following command to install prometheus components.
+Fadvisor use grafana to present cost estimates. Using following command to install a grafana.
 
 ```console
-helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-helm repo update
+helm repo add grafana https://grafana.github.io/helm-charts
+helm install grafana -f https://raw.githubusercontent.com/gocrane/helm-charts/main/integration/grafana/override_values.yaml -n crane-system --create-namespace grafana/grafana
```
Comment (mfanjie) [inline README.md:107]: is there a flag to set grafana service as NodePort or LoadBalancer by default?

File: README.md [hunk 3]
```diff
@@ -76,49 +76,311 @@ Crane is composed of the following components:
...
+```
+
+visit [Cost Report](http://127.0.0.1:3000/dashboards) here with account(admin:admin).
+
+### Analytics and Recommend Pod Resources
+
+Create an **Resource** `Analytics` to give recommendation for deployment: `craned` and `metric-adapter` as a sample.
+
+```console
+kubectl apply -f https://raw.githubusercontent.com/gocrane/crane/main/examples/analytics/analytics-resource.yaml
```
Comment (mfanjie) [inline README.md:179]: got a 404

File: README.md (hunk 2 above)
Comment (qmhu) [reply README.md:107]: Will address it in later pr

**Pull request merged**